lua/ttt2/libraries/modelbuilder.lua

```diff
@@ -0,0 +1,316 @@
+---
+-- Used to build custom world and view models. 
+-- @author Mineotopia
+-- @module modelbuilder
```

Comment:
Not much code was reused here, this is mostly done from scratch after learning how to do it. But you're right, linking this would probably be a good idea